src/PKSim.Core/Snapshots/Mappers/ParameterMapper.cs

```diff
@@ -88,7 +88,10 @@ private void updateParameterValue(SnapshotParameter snapshot, double parameterBa
          var snapshotValueInBaseUnit = parameter.ConvertToBaseUnit(snapshot.Value);
 
          if (!ValueComparer.AreValuesEqual(baseValue, snapshotValueInBaseUnit))
```

Comment:
Not convinced that this is required anymore. If the parameter is in the snapshot, then the value should be overwrittten. I don't think this is wrong however. TO discuss